### ISSUE TITLE
npm did not add, move, uninstall, or update any packages

### ISSUE THREAD

**Issue by davidklotz** [bdkjones/CodeKit#647]
**Quick, short summary:**
Trying to reinstall Premailer package with npm and it's not having it.

**Expected results:**
that it would install same as other packages.  
**Actual results:**
_npm did not add, move, uninstall, or update any packages.
_This happens when:
 - All installed packages are already up to date
 - There are no extraneous packages to remove
 - Your Mac is not connected to the Internet__

**Exact steps to reproduce:**
search package premailer, click install button. 
**A link to download a simplified project or file that shows the issue:**
N/A
**Your configuration (any details about your system that you think might be relevant)**
I've looked at a couple of past submitted issues, attempted uninstall and reinstall of Node and npm, and this hasn't seemed to solve the issue.  Mac node version v15.11.0, and npm 7.6.2.  I've also tried uninstalling Codekit and reinstalling but still get the same results with this package. 
![Screen Shot](https://user-images.githubusercontent.com/19672377/110550772-44419b00-8102-11eb-8d93-ffa2c88fe2af.jpg)

**Comment by bdkjones** [comment]
The reason is that you have npm 7.x installed on your Mac. If you look at the packages area, you’ll see that the packages WERE installed. Npm 7.x broke the JSON output foe the install command, so CodeKit doesn’t receive any results from npm and shows that message. I’ve filed a bug report against npm for this.

-Bryan

**Comment by davidklotz** [comment]
thanks for the quick response sorry I'm new to installing packages, but I took this as that if I roll back npm that it might work.  I rolled back to 6.14.11 and am getting a mix of things. If I install for example the package "animate.css" The node_modules folder is created and all goes as expected.  However, if I uninstall that, I now get "error determining installed..".  If I remove the package.json file and node_modules folder, then it says "no installed packages" as normal.  If I then attempt to install the premailer package, only the package.json file is created, and the node_modules folder is not and it does not show the packages as being installed in the packages area.  

So I just wanted to check with you if this is just related to the bug you mentioned or if you think something else is going on as well?

![Screen Shot](https://user-images.githubusercontent.com/19672377/110554556-0a27c780-8109-11eb-962b-49abf0c37dad.jpg)

![Screen Shot 2](https://user-images.githubusercontent.com/19672377/110554383-b321f280-8108-11eb-9d3f-3f997482b641.jpg)

![Screen Shot](https://user-images.githubusercontent.com/19672377/110554370-aac9b780-8108-11eb-9d89-b69cd7ffc3bd.jpg)

![Screen Shot](https://user-images.githubusercontent.com/19672377/110555796-39d7cf00-810b-11eb-8050-a61434a2cefc.jpg)
